### ISSUE TITLE
Newsletter

### ISSUE THREAD

**Issue by honzajavorek** [pyvec/zapojse#11]
Původní nápady:

> Máme e-mailové adresy ze srazů, ale nic na ně neposíláme. Přitom by stálo za to lidem alespoň připomínat, že bude další sraz a o čem bude.
> 
> Kdyby se tomu chtěl někdo dlouhodobě věnovat, mohl by dělat "výběr z toho nejlepšího, co se za poslední měsíc událo v Pythonu (v ČR)".
> 
> Byly tady nějaké záblesky chtění ze stran organizátorů pražského Pyva a taky existoval nějaký prototyp od Martina Putniorze, ale nic se nyní neděje.

Další:

> zpravodajstvi z ceskeho pythonu - rssko filtr na python / search napric twitter / vydane knizky v cestine / zdrojak tag python, root tag python, ... takove python hacker news
> 
> házelo by se to na Twitter, dalo by se to odebírat, mělo by to výstup na newsletter

Co já si teď myslím: Máme teď už http://blog.python.cz/, takže by newsletter mohl vycházet klidně jako článek na blogu a zároveň by se mohl rozeslat na maily.

**Comment by honzajavorek** [comment]
@starenka ses na tom chtěl nějak podílet myslim, tak tě tady jen tak náhodně pingnu

**Comment by frenzymadness** [comment]
Nápad je to skvělý, ale má to dvě ale:
- pro jedhono člověka to může být dost pracné (i když to částečně závisí na frekvenci)
- jeden člověk bude mít vždy jen omezené množství zdrojů, kam se dívá pravidelně nebo kam se podívá jen pro účely newsletteru

Co to udělat takto:
1. Založíme issue-only repo
2. Pro každé vydání newsletteru uděláme Issue
3. Budeme tam společně přispívat všichni, když kdokoli narazí na něco relevantního a zajímavého (zároveň se to tím stane ověřené, protože kdybych hledal sám články do newsletteru, možná bych některé z nich nečetl celé)
4. Až toho bude dost nebo po určitém čase se z toho udělá článek na blog & hromadný email.

Pokud toho bude dost, může to vycházet pravidelně. To by asi bylo dobré i s ohledem na propagaci srazů a kurzů a taky aby to byl opravdu **news**letter a nebyly tam neaktuální věci.

Co Vy na to?

**Comment by stlk** [comment]
@honzajavorek Zkusim udelat nejaky koncept a publikovat ho tady.

**Comment by kvbik** [comment]
mam pocit, ze tohle @lspdv a @zuzejk trosku rozjely, coz?

**Comment by honzajavorek** [comment]
No PyLadies (@zuzejk) rozjely vlastní mailing list a newsletter. Nějak jim to funguje. Mohli bychom se tím inspirovat a zavést to takto pro celý Python v ČR. Přijde mi, že se ten problém dá celkem snadno rozdělit na dvě části a to odráží i issues zde:

- sběr adres #39
- posílání nějakého obsahu #11

Sběr adres se musí udělat první a může fungovat nějakou dobu i bez toho, aby se cokoliv posílalo. Jakmile budeme mít vyřešený sběr, můžeme řešit jak třeba generovat newsletter z dostupných věcí (třeba z https://github.com/pyvec/python.cz/issues/220) a kdo to bude dělat.

**Comment by hroncok** [comment]
Já bych to dělal obráceně. Začít obsahem, vydávat "newsletter" na blogu a když se ukáže, že jsme schopní to pravidelně a hlavně dlouhodobě  tvořit, řešit rozesílání emailem. Newsletter představuje určitý závazek odběratelům, takže bych se do toho nehrnul, dokud neuvidíme, že to umíme psát.

**Comment by honzajavorek** [comment]
Zase ten mailing list je obecnější než newsletter. Jde i o pracovní nabídky atd. Slibem, že někomu něco v budoucnu pošleš, nezarmoutíš. A ve chvíli, kdy máš co poslat (třeba i jen to, že bude PyCon CZ), tak máš adresy, kam to můžeš poslat.

**Comment by lspdv** [comment]
V tomhle ma Miro pravdu. U PyLadies newslettery asi skoncily, dlouho jsem
zadny nikde nepotkala a je to skoda, protoze se lidem libil, jen asi neni
newsletterovej clovek.

**Comment by honzajavorek** [comment]
Nope, PyLadies newslettery jedou a newsletterovej človek je, je to @zuzejk. Mají ale přes prázdniny přestávku.

**Comment by lspdv** [comment]
Posledni prisel 19.6. a nikdo nevi ze jsou od nich jen prazdniny. :-)
pristi vikend je PyWorking (to vim ne z newsletteru, protoze to v zadnem
nebylo, ale protoze tam musim bejt za MSD)  a tak by se byval newsletter
hodil. Predpokladam, ze ucast na nem tak bude tentokrat o dost mensi. Holky
se me ptaji proc uz nejsou newslettery. To je vse co k tomu muzu rict a co
vim stejne jako odberatele. A tam je asi hacek

;-) Nope si kazdy nechme pro dobre pratele, dekuji.

**Comment by zuzejk** [comment]
(Když už to tady je jako konkrétní námitka, tak jen doplním, že účast na blížícím se PyWorkingu v MSD je dle organizátorky plná i bez newsletteru, info se rozesílalo mailem.)

A teď obecně k tématu na základě mých zkušeností s newsletterem pro pyladies:
Dost dobré by bylo, kdyby se mohl udělat nějaký automatický mail, který by se sám odeslal každý měsíc na mail organizátorům pravidelných aktivit a jejich odpověď (např co bude příští měsíc za téma Pyva, nebo za workshop + kdy a kde) by se někam uložila do archu/tabulky a odeslala by se mi jakožto tvůrci newsletteru kopie této odpovědi. Takže bych věděla, že to je ready. 
Pro nepravidelné a náhodně přispěvatele do newsletteru by pak bylo udělátko, kam by vepsali, co chtějí do newsletteru dát - opět by se to uložilo do toho archu a přišla by mi kopie mailem. (Takoví náhodní přispěvatelé se zatím u Pyladies neobjevují, na druhou stranu jsme ani nikoho nikterak k takovým činnostem nemotivovali, takže o tom spíš nikdo mimo organizátorky a kouče ani neví.) 
Různé tipy na akce / online kurzy / literaturu ohledně pythonu pro začátečníky se objevují hlavně na pyladies FB stránkách, takže v ideálním případě bych si to z FB mohla taky do toho archu nějak sosat.

Hlavní výhody tohoto návrhu: vyřešil by se hlavně problém s opakujícím se každoměsíčním sběrem informací od stejných lidí a navíc tím, že by odpovědi byly někde zaznamenány, tak v případě mé indispozice by si je mohl můj záskok snadno dohledat, nebo bych se mohla velmi jednoduše s někým střídat.

Tohle všechno je ale specifické pro náš pyladies newsletter, takže nevím, jestli se to nějak dá vztáhnout i na newsletter celokomunitní... Omlouvám se za dlouhý příspěvek. :)

**Comment by lspdv** [comment]
@zuzejk napad vyborny, vim jak na to za pomoci mailchimp. dokonce by melo
jednoduse zaridit, aby se podobny vyzvedny email posilal kazdy mesic uplne
sam.

Pokud chces muzu neco nahodit a vozkousime to mezi sebou.

Dne 28. srpna 2017 15:59 zuzejk <notifications@github.com> napsal(a):

**Comment by honzajavorek** [comment]
Tak minimálně pokud udelame tu stránku s nejbližšími eventy na Python.cz,
tak by to mělo zjednodušit sběr některých tech informaci, snad.

**Comment by frenzymadness** [comment]
Bezva nápad. Klidně ten výzvědný mail posílejte i mi.

PyLadies newsletter jsem nikdy neviděl a nevím jak moc je specifický pro Prahu, ale celkem by se mi líbilo, kdyby se do něj přidaly i věci týkající se ostatních a seznam odresátů by se rozšířil na celou republiku. Vycházím z toho, že když už se s tím někdo dělá, tak ať to má co nejvíce čtenářů, ale jak říkám, pokud je to hodně Praha-specifik, tak to asi nebude reálné.

Nebo, co takhle z toho udělat jeden společný i pro zbytek komunity mimo PyLadies a tím ještě zvětšit základnu čtenářů? Jak moc blbý a pracný je to nápad?
